stories/utils.js

```diff
@@ -22,6 +22,10 @@ export const previewWrapper = ({
       </li>
     </ol>
   `,
+  filters = `
+    <!-- @TODO: replace with a RefinementList  -->
+    <ais-menu attribute="brand" />
```

Comment:
nice idea, but why does it even need a default?

Reply:
Mainly because in a couple of widgets like `CurrentRefinements`, `ClearRefinements` even `Hits` it's interesting to have more refinement capabilities to test with.